src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy

```diff
@@ -37,10 +37,17 @@ class ComposeDown extends DefaultTask {
     VersionNumber getDockerComposeVersion() {
         new ByteArrayOutputStream().withStream { os ->
             project.exec { ExecSpec e ->
-                e.commandLine 'docker-compose', '--version'
+                e.commandLine prepareCommand(['docker-compose', '--version'])
                 e.standardOutput = os
             }
             VersionNumber.parse(os.toString().trim().findAll(/(\d+\.){2}(\d+)/).head())
         }
     }
+
+    protected Iterable<String> prepareCommand(List<String> baseCommand) {

```

Comment:
Is it required to change commands in `ComposeDown`? I think that `docker-compose` _connects_ particular commands on `project` base - it's name of directory (and can be set by `-p` argument). So I think that we don't have to change `ComposeDown`, we should change `build` and `up` calls only.

Could you please verify this assumption?